recipe/conda_build_config.yaml

```diff
@@ -485,8 +485,7 @@ libxml2:
 libuuid:
   - 2.32.1
 lz4_c:
-  - 1.8.1    # [not (aarch64 or ppc64le)]
-  - 1.8.3    # [aarch64 or ppc64le]
+  - 1.8.3
```

Comment:
While I'm OK with this I don't know enough about the `aarch64` and `ppc64le` situation to comment. @mariusvniekerk is probably a better person to merge this one.